Doxyfile

```diff
@@ -1980,7 +1980,7 @@ INCLUDE_FILE_PATTERNS  =
 # recursively expanded use the := operator instead of the = operator.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-PREDEFINED             =
+PREDEFINED             = _DOXYGEN_
```

Comment:
C++ spec 17.4.3.1.2. These types of names are reserved for the compiler.  I would simply remove the underscores and make the defined name a bit more specific so it doesn't clash.  So something like `EOSIO_CDT_DOXYGEN`.